### ISSUE TITLE
Reproduced the experimental results on CIFAR-10?

### ISSUE THREAD

**Issue by jalused** [keunwoochoi/residual_block_keras#7]
Thanks for your great work, I tried to reproduce the experimental results of residual network on CIFAR-10, I followed the architecutre described in section 4.2 in Kaiming He's paper http://arxiv.org/pdf/1512.03385.pdf, a 20-layer residual network, used sgd with 0.1 lr, 0.9 momentum, but it diverges soon. I checked my code, but I didn't firgure out what mistake I have made. So I just wonder have you reproduced the result on CIFAR-10, if so, could you please share your code and the pre-trained model? Thanks so much.

**Comment by keunwoochoi** [comment]
Hi, I'll take a look. I just extended the example code to cifar-10 but the network is not the same as Kaiming He's one. But would you first check the initialisation scheme? Otherwise starting with smaller learning rate might work.

**Comment by jalused** [comment]
Hi, @keunwoochoi, I have reproduced the result on cifar-10 using your code, I made some mistake building the network. Thanks for your great work!

**Comment by keunwoochoi** [comment]
Great! Is it the same structure of Kaiming? Then it would be nice if you could share the code.
